test/e2e/spec/api_spec.js

```diff
@@ -127,46 +151,813 @@ describe("API", function () {
 
             });
 
-            describe("domains endpoint", function () {
+            describe("on domains/ endpoint", function () {
 
                 it("can register a domain name", function () {
                     var domain = 'e2etest-' + require("uuid").v4() + '.dedyn.io';
                     return expect(chakram.post('/domains/', {'name': domain})).to.have.status(201);
                 });
 
+                it("has correct JSON schema");
```

Comment:
No. That's a "pending test" (neither failing nor passing); shows up with "-" instead of "x" or checkmark in the results. It's meant to say that this test has no implementation yet.